icons/.svgrrc.js

```diff
@@ -0,0 +1,22 @@
+const path = require('path');
```

Comment:
Is this file named correctly or is there an extra r in it?

Reply:
It should have two rr's: `svgr` (the package) and `rc` (run commands)